libs/langchain/pyproject.toml

```diff
@@ -149,6 +149,7 @@ msal = {version = "^1.25.0", optional = true}
 databricks-vectorsearch = {version = "^0.21", optional = true}
 couchbase = {version = "^4.1.9", optional = true}
 dgml-utils = {version = "^0.3.0", optional = true}
+tritonclient = {version = "^2.34.0", optional = true}
```

Comment:
out of curiosity: should we bump to 2.40.0?

Reply:
Yes, 2.40.0 was published on December 1st so would make since to use it I would assume. I don't see any known bugs with it around the features we are using

Reply:
@jdye64 another question -> is there a strict need to both support the GRPC and HTTP clients? We are deprecating a class level `streaming` argument in new LLM implementations in favor of streaming for `.stream()/.astream()` methods and batch/single calls in `.batch()/.abatch()` calls.

Since we always want to support streaming, I'd imagine we'd need the grpc client then so we could only use that?